portfolio/src/main/webapp/index.html

```diff
@@ -8,13 +8,65 @@
   </head>
   <body>
     <div id="content">
-      <h1>Chenyu Zhang</h1>
-      <img src="/images/avatar.JPG" height="100"/>
-      <p>I am an undergraduate student at Johns Hopkins University. 
+      <div id="header-container">
+        <div id="avatar">
+          <img src="/images/avatar.JPG" alt="Me"/>
+          <img src="/images/avatar2.JPG" alt="Me2" class="img-top"/>
+        </div>
+        <h1 id="name">Chenyu Zhang</h1>
+        <p id="intro">I am an undergraduate student at Johns Hopkins University. 
+          I was born in Nanjing, China, and I currently live in Baltimore, Maryland, USA.
           This portfolio webpage is written during my internship at Google in summer 2020.</p>
+      </div>
       
-      <p>LinkedIn page: <a href="https://www.linkedin.com/in/chenyu-heidi-zhang-jhu/">link</a></p>
-      <p>GitHub page: <a href="https://github.com/ChenyuHeidiZhang">link</a></p>
+      <p>At JHU, my majors are Applied Math, Computer Science, and Cognitive Science. 
+        I enjoy all three of my majors and I think they complement each other very well.
+        Aside from school, I am part of the Blue Orchid Dance Club, where we practice trational Chineses dances and give performances at various events 
+        (which are, sadly, suspended due to COVID-19). 
+        I am also a research assistant at the Mind Brain Institute, and a software developer on the Otoglobal Health design team. 
```

Comment:
This is really cool. For your real final portfolio I would recommend adding more details about these experiences you have since they're related to your major. You want to highlight your skills :)

Reply:
Ok I expanded on that. (Now I think it's a bit too much...) Let me know what you think when I push : )